playwright-e2e/pages/base-page.ts

```diff
@@ -1,18 +1,21 @@
-import { type Page, type Locator } from "@playwright/test";
-export class BasePage {
+import { type Page, type Locator, expect } from "@playwright/test";
+
+export abstract class BasePage {
   readonly nextStep: Locator;
   readonly go: Locator;
-  readonly page: Page;
-  readonly saveAndContinue: Locator;
+  readonly page: Page ;
+  readonly saveAndContinue : Locator;
   readonly continueButton: Locator;
+  readonly signOut: Locator;
   readonly checkYourAnswersHeader: Locator;
 
-  constructor(page: Page) {
-    this.page = page;
-    this.nextStep = page.getByLabel("Next step");
-    this.go = page.getByRole("button", { name: "Go" });
-    this.saveAndContinue = page.getByRole("button", { name: "Save and continue" });
```

Comment:
I think should revert the formatting back to these to make sure the spacing is uniform! (Although they should be single quotes from your PR!)